### PR TITLE
[back8sup] add expire value for old backups

### PR DESCRIPTION
**Pull request by eyenx**
Adds the Expire value for deleting old backups.

See https://github.com/adfinis-sygroup/back8sup/issues/9

* [x] Merge https://github.com/adfinis-sygroup/back8sup/pull/10

### CODE REVIEW

File: charts/back8sup/Chart.yaml [hunk 1]
```diff
@@ -2,7 +2,15 @@ apiVersion: v2
...
-version: 0.1.1
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 0.1.2
+
+# This is the version number of the application being deployed. This version number should be
+# incremented each time you make changes to the application. Versions are not expected to
+# follow Semantic Versioning. They should reflect the version the application is using.
```
Comment (hairmare) [inline charts/back8sup/Chart.yaml:13]:
```suggestion
version: 0.1.2
```

**Comment by hairmare** [comment]
You might consider versioning back8sup to we can use the `appVersion` field to know which version this is compatible with.

This should most likely be held back until https://github.com/adfinis-sygroup/back8sup/issues/9 is merged.

**Comment by eyenx** [comment]
`REBASED`

**Comment by eyenx** [comment]
pls review @hairmare https://github.com/adfinis-sygroup/back8sup/issues/10  has been merged

File: charts/back8sup/values.yaml [hunk 2]
```diff
@@ -12,7 +12,7 @@ image:
   # image.pullPolicy -- set the image pullPolicy
   pullPolicy: IfNotPresent
   # image.tag -- set the tag of the image
-  tag: "latest"
+  tag: "v0.1.3"
```
Comment (hairmare) [inline charts/back8sup/values.yaml:15]: Did you consider using just the appVersion like other charts do?

Comment (eyenx) [reply charts/back8sup/values.yaml]: see https://github.com/adfinis-sygroup/helm-charts/pull/43/commits/9daa637056e34622e14c12e40c2741b5cd8635b9

File: charts/back8sup/values.yaml [hunk 3]
```diff
@@ -12,7 +12,9 @@ image:
   # image.pullPolicy -- set the image pullPolicy
   pullPolicy: IfNotPresent
   # image.tag -- set the tag of the image
-  tag: "v0.1.3"
+  # Specify a tag to override which version of timed to deploy.
+  # If no tag is specified the appVersion from Chart.yaml is used as tag.
+  # tag:
```
Comment (hairmare) [inline charts/back8sup/values.yaml:17]:
```suggestion
  # Specify a tag to override which version of timed to deploy.
  # If no tag is specified the appVersion from Chart.yaml is used as tag.
  tag: ""
```

The caasperli chart example does it like this:
```
  # image.tag -- Overrides the image tag whose default is the chart version.
  tag: ""
```

Comment (eyenx) [reply charts/back8sup/values.yaml:17]: Well cool. But not the `charts/timed` took it from there.

**Pull request merged**